README.md

```diff
@@ -16,15 +15,27 @@
 }
 ```
 
-5. Add a script to your package.json to lint your code:
+4. Add a script to your package.json to lint your code:
 
 ```
 "scripts": {
-    "lint": "eslint ./src",
+    "lint": "eslint ./",
 },
 ```
 
-6. Now you can manually lint your code by running yarn run lint
+5. The `twig` eslint config uses [eslint-config-compat](https://www.npmjs.com/package/eslint-plugin-compat). This will lint your code for browser compatibility. There are default settings but we recommend adding these settings to your package.json file for Twig Science projects:
+
+```
+"browserslist": [
+    "last 2 versions",
+    "not ie <= 11",
+    "ios_saf >= 12",
```

Comment:
Shouldn't this be 13.4 as we discussed?
<img width="800" alt="image" src="https://github.com/TwigWorld/eslint-config-twig/assets/93528601/473e2832-0d6f-42e9-8f51-173b2e2c31c9">

Reply:
See the comment below from Michael. After conversations with Kati we're planning to include IOS >= 14.